src/java.desktop/macosx/classes/com/apple/laf/AquaScrollBarUI.java

```diff
@@ -188,11 +189,17 @@ public void layoutContainer(final Container fScrollBarContainer) {
     }
 
     protected Rectangle getTrackBounds() {
-        return new Rectangle(0, 0, fScrollBar.getWidth(), fScrollBar.getHeight());
+        Insets insets = fScrollBar.getInsets();
+        return new Rectangle(insets.left, 0,
+                fScrollBar.getWidth() - (insets.left + insets.right),
+                fScrollBar.getHeight());
```

Comment:
What about vertical insets? Horizontal insets are handled now yet the border could render something vertically.